KebPOS/Services/ProductService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace KebPOS.Services
+{
+    internal class ProductService
+    {
+        public List<string> GetProducts()
+        {
+            var order = new List<string>();
```

Comment:
The products list needs to be a List of the type Product, which can be found in the models folder. 😊